packages/forms/resources/views/components/text-input.blade.php

```diff
@@ -28,11 +29,13 @@
         :suffix-actions="$suffixActions"
         :suffix-icon="$suffixIcon"
         :valid="! $errors->has($statePath)"
+        :is-revealable="$isRevealable"
         class="fi-fo-text-input"
         :attributes="
             \Filament\Support\prepare_inherited_attributes($getExtraAttributeBag())
                 ->class(['overflow-hidden'])
         "
+        x-data="{inputType: '{{ $getType() }}'}"
```

Comment:
I think we should only render this when revealable.

Reply:
Is rendering by `\Illuminate\Support\Js` be okay? Like this:

```php
:x-data="$isRevealable ? \Illuminate\Support\Js::from(['inputType' => $getType()]) : null"
```

Reply:
We don't need it to be an Alpine component at all when not revealable.

Reply:
Yes, it doesn't render as Alpine component, `x-data` is not present at all this way